src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2008-2018 TrinityCore <https://www.trinitycore.org/>
+ * Copyright (C) 2008-2017 TrinityCore <http://www.trinitycore.org/>
```

Comment:
Copyright 😋

Reply:
Yeah, I should have updated the Copyright end year in the TC branch `3.3.5-trial_of_the_champion`.
Thanks for the reminder. ^^